<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdditionalSettingsDialog</class>
 <widget class="QDialog" name="AdditionalSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Additional Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxPosition">
     <property name="title">
      <string>Position Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayoutPosition">
      <item row="0" column="0">
       <widget class="QLabel" name="labelPosition">
        <property name="text">
         <string>Position (X, Y):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayoutPosition">
        <item>
         <widget class="QSpinBox" name="spinBoxX">
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelComma">
          <property name="text">
           <string>,</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxY">
          <property name="minimum">
           <number>-9999</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCurrentPos">
        <property name="text">
         <string>Use Current Position:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBoxCurrentPos">
        <property name="text">
         <string>Current mouse position</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTiming">
     <property name="title">
      <string>Timing Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayoutTiming">
      <item row="0" column="0">
       <widget class="QLabel" name="labelHoldDuration">
        <property name="text">
         <string>Hold Duration (ms):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxHoldDuration">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelHoverDuration">
        <property name="text">
         <string>Hover Duration (ms):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxHoverDuration">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAction">
     <property name="title">
      <string>Action Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayoutAction">
      <item row="0" column="0">
       <widget class="QLabel" name="labelClickCount">
        <property name="text">
         <string>Click Count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxClickCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelKeyName">
        <property name="text">
         <string>Key Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditKeyName">
        <property name="placeholderText">
         <string>Enter key name (e.g., A, Space, Enter)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AdditionalSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdditionalSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCurrentPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxX</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCurrentPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxY</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>